### ISSUE TITLE
 error  Prop type `array` is forbidden

### ISSUE THREAD

**Issue by yonghuming** [processing/p5.js-web-editor#1548]
```
ERROR in ./client/modules/IDE/components/NewFileForm.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/IDE/components/NewFileForm.jsx
  52:5  error  Prop type `object` is forbidden  react/forbid-prop-types

✖ 1 problem (1 error, 0 warnings)
ERROR in ./client/modules/IDE/components/NewFolderForm.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/IDE/components/NewFolderForm.jsx
  54:5  error  Prop type `object` is forbidden  react/forbid-prop-types

✖ 1 problem (1 error, 0 warnings)
ERROR in ./client/modules/IDE/pages/IDEView.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/IDE/pages/IDEView.jsx
  481:5  error  Prop type `array` is forbidden  react/forbid-prop-types
  516:5  error  Prop type `array` is forbidden  react/forbid-prop-types

✖ 2 problems (2 errors, 0 warnings)
ERROR in ./client/modules/User/components/APIKeyForm.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/User/components/APIKeyForm.jsx
  11:3  error  Prop type `object` is forbidden  react/forbid-prop-types
  12:3  error  Prop type `object` is forbidden  react/forbid-prop-types

✖ 2 problems (2 errors, 0 warnings)
ERROR in ./client/modules/User/components/AccountForm.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/User/components/AccountForm.jsx
  106:5  error  Prop type `object` is forbidden  react/forbid-prop-types
  107:5  error  Prop type `object` is forbidden  react/forbid-prop-types
  108:5  error  Prop type `object` is forbidden  react/forbid-prop-types
  109:5  error  Prop type `object` is forbidden  react/forbid-prop-types

✖ 4 problems (4 errors, 0 warnings)
ERROR in ./client/modules/User/components/LoginForm.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/User/components/LoginForm.jsx
  47:5  error  Prop type `object` is forbidden  react/forbid-prop-types
  48:5  error  Prop type `object` is forbidden  react/forbid-prop-types

✖ 2 problems (2 errors, 0 warnings)
ERROR in ./client/modules/User/components/NewPasswordForm.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/User/components/NewPasswordForm.jsx
  46:5  error  Prop type `object` is forbidden  react/forbid-prop-types
  47:5  error  Prop type `object` is forbidden  react/forbid-prop-types

✖ 2 problems (2 errors, 0 warnings)
ERROR in ./client/modules/User/components/ResetPasswordForm.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/User/components/ResetPasswordForm.jsx
  35:5  error  Prop type `object` is forbidden  react/forbid-prop-types

✖ 1 problem (1 error, 0 warnings)
ERROR in ./client/modules/User/components/SignupForm.jsx
Module Error (from ./node_modules/_eslint-loader@2.2.1@eslint-loader/index.js):

/root/p5.js-web-editor/client/modules/User/components/SignupForm.jsx
  74:5  error  Prop type `object` is forbidden  react/forbid-prop-types
  75:5  error  Prop type `object` is forbidden  react/forbid-prop-types
  76:5  error  Prop type `object` is forbidden  react/forbid-prop-types
  77:5  error  Prop type `object` is forbidden  react/forbid-prop-types

✖ 4 problems (4 errors, 0 warnings)
```
![image](https://user-images.githubusercontent.com/435079/90091517-31780b00-dd59-11ea-9cc3-e23e9477cbd1.png)

- Steps to reproduce this bug:
On Chrome Ubuntu 16.04
the errors occured after npm run

**Comment by catarak** [comment]
Thanks for reporting! Which version of Node/React are you using? For me, in development, I'm not seeing these errors, but I think when eventually libraries and Node get updated, this will need to be fixed.

**Comment by vulongphan** [comment]
Hi I am using React v16.14.0 and Node v14.5.0. I solve `Prop type 'object' is forbidden` by using `PropTypes.objectOf(PropTypes.shape())` instead of `PropTypes.object.isRequired`.

Hope it help.

**Comment by catarak** [comment]
Okay! This project is currently using React `v16.12.0` and Node `v12.6.1`, so I think this will come up when updating React/Node.

**Comment by KKVANONYMOUS** [comment]
@catarak I guess this issue has been solved by #1692

**Comment by catarak** [comment]
thanks for pointing that out!
